rosdep/base.yaml

```diff
@@ -2408,6 +2411,18 @@ libgstreamer1.0-dev:
   fedora: [gstreamer1-devel]
   gentoo: ['media-libs/gstreamer:1.0']
   ubuntu: [libgstreamer1.0-dev]
+libgstrtspserver-1.0-0:
+  debian: [libgstrtspserver-1.0-0]
+  ubuntu: [libgstrtspserver-1.0-0]
+libgstrtspserver-1.0-0-dbg:
+  debian: [libgstrtspserver-1.0-0-dbg]
+  ubuntu: [libgstrtspserver-1.0-0-dbg]
```

Comment:
We don't typically add the `-dbg` packages to rosdep keys; it's just not something that downstream consumers are going to depend on.  So please remove this.

Reply:
Makes sense, will fix.